components/gc-follow-us/followus.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @title WET-BOEW Follow us component
+ * @overview Plugin used to replace Twitter with "X" - Deprecated
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @garneauma
+ */
+( function( $, window, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "followus",
+	selector = "." + componentName,
+	initEvent = "wb-init " + selector,
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var elm = wb.init( event, componentName, selector ),
+			$elm;
+
+		if ( elm && event.currentTarget === event.target ) {
+			$elm = $( elm );
+
+			elm.querySelector( ".twitter .wb-inv" ).innerHTML = "X";
+
+			// Identify that initialization has completed
+			wb.ready( $elm, componentName );
```

Comment:
Considering we use the $elm only once, let's do the jQuery encapsulation only where it needed.

```suggestion
			;

		if ( elm && event.currentTarget === event.target ) {

			elm.querySelector( ".twitter .wb-inv" ).innerHTML = "X";

			// Identify that initialization has completed
			wb.ready( $( elm ), componentName );
```